<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddHeir</class>
 <widget class="QWidget" name="AddHeir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="heir_LE">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="detected_LA">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_LE">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="securityAnswer_LE">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addHeir_PB">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add heir</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 72, 72);
</string>
   </property>
   <widget class="QLabel" name="securityQuestion_LA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LogIn_LA">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 255, 255, 210);</string>
    </property>
    <property name="text">
     <string>Add heir</string>
    </property>
   </widget>
   <widget class="QLabel" name="addUsername_LA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Enter heir's username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="enterPass_LA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Enter your password:</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>heir_LE</zorder>
  <zorder>detected_LA</zorder>
  <zorder>password_LE</zorder>
  <zorder>securityAnswer_LE</zorder>
  <zorder>addHeir_PB</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
